### PR TITLE
Optimizing Connections

### LINKED ISSUE #40: Optimize the algorithm that would update daily menu of restaurants
Optimize the script on the branch restaurant-data-mysql that has a name update_menu.py
Look into minimizing connections with MySQL database in that script to only one.

### PR DESCRIPTION
**Pull request by jandvorak-dot**
Optimized to use only one connection.

close #40

### CODE REVIEW

File: update_menu.py [hunk 1]
```diff
@@ -33,30 +32,31 @@ def get_menu(restaurant_id):
     return data
 
 
-def update_menu_database(res_id, menu):
-    cnx = mysql.connector.connect(user='script', password='LetMeIn:)123',
-                                  host='127.0.0.1',
-                                  database='restaurant_recommender')
-    cursor = cnx.cursor()
+def update_menu_database(res_id, menu, cnx, crs):
+    cursor = crs
```
Comment (ilyassakhanov) [inline update_menu.py]: why is this line necessary?

**Review by ilyassakhanov — COMMENTED** [review]
Overall the goal was achieved, change that strange variable assignment you have and it should be good for merging

File: update_menu.py (hunk 1 above)
Comment (jandvorak-dot) [reply update_menu.py]: Not really necessary. Changed it.

**Review by ilyassakhanov — APPROVED** [review]
That looks good
